### PR TITLE
Fix the GCDepth graph and make it look more like the gnuplot version

### PR DESCRIPTION
**Pull request by jenniferliddle**

### CODE REVIEW

**Comment by mgcam** [comment]
As far as I know, a json version is not required; the pull request should wait for David to consider. Please do not merge.

File: demo/index.html [hunk 1]
```diff
@@ -3,16 +3,25 @@
...
+            border-radius: 15px 15px 15px 15px; 
+            overflow: hidden; 
+            font-size: 14px; 
+          } 
+        </style>
         <script data-main="main" src="/external/requirejs/require.js"></script>
     </head>
     <body>
-        <div id="instructions">
-            <p>To switch to another set of graphs, use the &quot;.&quot; key to move forward and the &quot;,&quot; key to move backwards.</p>
```
Comment (mgcam) [inline demo/index.html]: No way to quickly and easily to look through large number of stats file, which is needed to validate that correct images are produced for difefrent input data.

**Comment by mgcam** [comment]
Is src/qcjson/gcdepth.js still needed? If not, both the module and its test should be removed.

File: demo/index.html (hunk 1 above)
Comment (jenniferliddle) [inline demo/index.html]: That's not the purpose of a demonstration program.

If such a program is needed, then tell me the requirements and use cases, and I can produce one.

**Comment by jenniferliddle** [comment]
src/qcjson/gcdepth.js is probably not needed at the momentm but it conceivably could be useful sometime. I'm reluctant to simple delete it. If we accept it into this pull request and then delete it, we could at least resurrect it if required at some future time.

**Comment by mgcam** [comment]
Replaced by https://github.com/wtsi-npg/bcviz/pull/64

**Pull request closed without merging**